WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -48,6 +50,9 @@
             </td>
           <% end %>
 
+          <td>
+            <%= registration.events.length %>
```

Comment:
Since we have the separate table for registration events the most optimal way to compute the registration events count is `registration.registration_events.count`. This will grab the count directly from database, whereas what you've done will grab the whole events from database then map them with Event objects and finally get length of the array.